examples/browser/package.json

```diff
@@ -10,7 +10,7 @@
     "monaco-languages": "^0.7.0",
     "monaco-typescript": "^2.1.2",
     "reflect-metadata": "^0.1.10",
-    "theia": "file:../../../Theia"
+    "theia": "file:../../../theia"
```

Comment:
Yes, but we can take care of that, no? We should assume the defaults going forward. So I'd like to merge these changes.